opm/output/eclipse/RestartValue.cpp

```diff
@@ -96,6 +96,18 @@ namespace Opm {
         this->addExtra(key, UnitSystem::measure::identity, std::move(data));
     }
 
+    void RestartValue::addExtra(const std::string& key, std::vector<float> data) {
+        std::vector<double> ddata(data.size());
+        std::copy(data.begin(), data.end(), ddata.begin());
+        this->addExtra(key, UnitSystem::measure::identity, std::move(ddata));
```

Comment:
You don't need construct+copy+move here.  You could just as well write this as
```C++
this->addExtra(key, std::vector<double>(data.begin(), data.end()));
```

Reply:
done